homeassistant/components/homeassistant_hardware/strings.json

```diff
@@ -72,7 +70,9 @@
         "fw_install_failed": "{firmware_name} firmware failed to install, check Home Assistant logs for more information."
       },
       "progress": {
-        "install_firmware": "Please wait while {firmware_name} firmware is installed to your {model}, this will take a few minutes. Do not make any changes to your hardware or software until this finishes."
+        "install_firmware": "Installing {firmware_name} firmware. Do not make any changes to your hardware or software until this finishes.",
```

Comment:
@matthiasdebaat should I remove the warning too?

> Do not make any changes to your hardware or software until this finishes.

You didn't include it in your design.

Reply:
This is what you designed:

<img width="608" height="244" alt="image" src="https://github.com/user-attachments/assets/b3539ffb-e9c9-4cc4-9dd2-eb18920430d4" />

Reply:
If you interrupt the firmware update the adapter will have corrupted firmware. The bootloader can usually resurrect it but I'd rather not take any chances.

Reply:
Lets keep the warning in this case.